crates/ursa-network/src/config.rs

```diff
@@ -34,6 +34,11 @@ pub struct NetworkConfig {
     pub identity: String,
     /// Keystore path. Defaults to ~/.ursa/keystore
     pub keystore_path: PathBuf,
+    /// Temporary HTTP tracker url. This is used for pre-consensus node announcements.
+    /// Defaults to devnet tracker.
+    pub tracker: String,
+    /// Prometheus metrics port
+    pub metrics_port: Option<u16>,
```

Comment:
If a user needs to change the metrics port, they would need to update in 2 places in config file?